bin/varnishstat/varnishstat_options.h

```diff
@@ -59,7 +51,7 @@
 	)
 
 STAT_OPT_1
-STAT_OPT_f
+VSC_OPT_f
```

Comment:
What you see as a breaking change is visible here has having no effect on `varnishstat`. Instead of a private `STAT_OPT` option this is now a public `VSC_OPT`.